lib/scrabble.rb

```diff
@@ -1,7 +1,29 @@
 class Scrabble
 
-  def score(word)
-    1
+  attr_reader :word, :scoring
+
+  def initialize(word)
+    @word = word
+    @scoring = []
+  end
+
+
+  def score
+    #split all the words into characters...=array
+    #for each in array, access the value
+    #add the values and return the score.
+    if @word.length == 1
+      @scoring = point_values[@word.upcase]
+    elsif @word == nil
+      @scoring = 0
+    else
+      words = []
+      words << @word.split("")
+      @scoring = @words.map do |letter|
```

Comment:
When you call .split, it returns an array. When you shovel it in to words, which is already an array you end up with a nested array.  You might want to save the return result like this `letters = @word.split`